tools/data-handler/test/template.test.ts

```diff
@@ -80,16 +69,13 @@ describe('template', () => {
     });
     const project = template.templateProject;
     const cards = await project.cards();
-    const templateCards = await template.cards();
+    //const templateCards = await template.cards();
```

Comment:
Is this something that should either be fixed, or removed?

Reply:
Fixed in [811d7c2](https://github.com/CyberismoCom/cyberismo/pull/127/commits/811d7c26427bc63e714c5db054551d000e028a08)